3723/1/represent_page.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'sorted_pages'
+
+class RepresentPages
```

Comment:
Moreover, all that do this class is printing, so it might be `PagePrinter`. Look at this:
```ruby
PagePrinter.new(rows).call
```
1. We've got a printer for pages
2. We create a new instance of the printer with some rows
3. We call the printer to do its stuff
???
n. PROFIT!

That's all about clarity. Final usage of the class might be clearly readable from the first sight

Reply:
Done